awsconfigs/apps/pipeline/base/kustomization.yaml

```diff
@@ -7,5 +7,5 @@ bases:
 images:
   - name: gcr.io/ml-pipeline/api-server
     newName: public.ecr.aws/c9e4w0g3/ml-pipeline/api-server
-    newTag: 2.0.0-alpha.3
+    newTag: 2.0.0-alpha.5
```

Comment:
we dont have this image right now, @akartsky  will update this later. keep it to .3 for now

Reply:
only this file or the rest of the pipeline yaml files as well?

Reply:
Only this image

public.ecr.aws/c9e4w0g3/ml-pipeline/api-server

Reply:
so in the helm charts they will be alpha.5?

Reply:
I mean both helm and kustomize but only this image because .5 does not exist as of now

All other images same as upstream

Basically, build it with public.ecr.aws/c9e4w0g3/ml-pipeline/api-server:2.0.0-alpha3

Reply:
Now should be good. deployment tag is alpha.3 while the rest follow upstream to be alpha.5